Standards/scs-XXXX-vN-Availability-Zones-Standard.md

```diff
@@ -0,0 +1,196 @@
+---
+title: Availability Zones Standard
+type: Standard
+status: Draft
+track: IaaS
+---
+
+## Introduction
+
+On the IaaS-Level especially in OpenStack it is possible to group resources in Availability Zones.
+Such Zones often are mapped to the physical layer of a deployment, such as e.g. physical separation of hardware or redundancy of power circuits or fire zones.
+But how CSPs apply Availability Zones to the IaaS Layer in one deplyoment may differ widely.
+Therefore this standard will address the minimal requirements that need to be met, when creating Avaiability Zones.
+
+## Terminology
+
+| Term               | Explanation                                                                                                                              |
+| ------------------ | ---------------------------------------------------------------------------------------------------------------------------------------- |
+| Availability Zone  | (also: AZ) internal representation of physical grouping of service hosts, which also lead to internal grouping of resources.             |
+| Fire Zone          | A physical separation in a data center that will contain fire within it. Effectively stopping spreading of fire.                         |
+| PDU                | Power Distribution Unit, used to distribute the power to all physical machines of a single server rack.                                  |
+| Compute            | A generic name for the IaaS service, that manages virtual machines (e.g. Nova in OpenStack).                                             |
+| Network            | A generic name for the IaaS service, that manages network resources (e.g. Neutron in OpenStack).                                         |
+| Storage            | A generic name for the IaaS service, that manages the storage backends and virtual devices (e.g. Cinder in OpenStack).                   |
+
```

Comment:
```suggestion
| CSP                | Cloud Service Provider, provider managing the OpenStack infrastructure.                                                              |

```

I added this into the glossary of most of my standards because I frequently received feedback that this might not be a well-established abbreviation outside of SCS yet. Maybe add it here as well?